src/external-apis/clearbit.ts

```diff
@@ -10,12 +10,16 @@ export async function getTwitterHandle(domain: string): Promise<null | string> {
     throw new Error('Do not call when running tests')
   }
 
-  const response = await axios.get(`https://company.clearbit.com/v2/companies/find?domain=${domain}`, {
-    headers: {
-      Authorization: `Bearer ${process.env.CLEARBIT_SECRET_API_KEY}`
-    }
-  })
+  try {
+    const response = await axios.get(`https://company.clearbit.com/v2/companies/find?domain=${domain}`, {
+      headers: {
+        Authorization: `Bearer ${process.env.CLEARBIT_SECRET_API_KEY}`,
+      },
+    })
 
-  const twitterHandle = response.data.twitter?.handle
-  return twitterHandle && `@${twitterHandle}`
+    const twitterHandle = response.data.twitter?.handle
+    return twitterHandle && `@${twitterHandle}`
+  } catch (err) {
+    return console.error(err), null
```

Comment:
Yeah I'm using the [comma operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Comma_Operator) to return null here. Since this is an `async` function that will be `Promise.resolve(null)`

The reason why is to elide the `tslint:no-expression-statement`